### PR TITLE
microprofile-health-118 Clarify the official name of the health spec …

### PR DESCRIPTION
**Pull request by antoinesd**
…and use it consistently

Also renamed artefact. Not sure it's expected

Signed-off-by: Antoine Sabot-Durand <antoine@sabot-durand.net>

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@
 // limitations under the License.
 //
 
-# Service Healthchecks
+# Microprofile Health Check
```
Comment (Ladicek) [inline README.adoc]: "Microprofile" -> "MicroProfile", capital P

File: spec/src/main/asciidoc/microprofile-health-spec.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@
 // limitations under the License.
 //
 
-= Service Health Checks
+= Microprofile Health Check
```
Comment (Ladicek) [inline spec/src/main/asciidoc/microprofile-health-spec.adoc]: "Microprofile" -> "MicroProfile", capital P

File: spec/src/main/asciidoc/overview.adoc [hunk 3]
```diff
@@ -15,7 +15,7 @@
 // limitations under the License.
 //
 
-= Service Health checks
+= Microprofile Health Check
```
Comment (Ladicek) [inline spec/src/main/asciidoc/overview.adoc]: "Microprofile" -> "MicroProfile", capital P

File: api/pom.xml [hunk 4]
```diff
@@ -23,8 +23,8 @@
         <relativePath>../</relativePath>
     </parent>
 
-    <artifactId>microprofile-health-api</artifactId>
-    <name>MicroProfile Health API</name>
+    <artifactId>microprofile-health-check-api</artifactId>
```
Comment (Ladicek) [inline api/pom.xml]: I wouldn't be a fan. Consistency is good, but not breaking users is better.

File: README.adoc (hunk 1 above)
Comment (antoinesd) [reply README.adoc]: thanks

File: api/pom.xml (hunk 4 above)
Comment (antoinesd) [reply api/pom.xml]: Yes, that's the reason why I asked on the ML. I'll probably revert that

**Comment by Emily-Jiang** [comment]
I think after the discussion on the mailing list, the agreed name should be MicroProfile Health since the group id and artifactid should not be updated for a number of good reasons.

**Comment by antoinesd** [comment]
@Emily-Jiang from what I read in the thread the agreement was on Health Check. I only reverted the artefacts name to avoid trouble. We should probably ask again on the ML

**Comment by antoinesd** [comment]
Eclipse MicroProfile Health is the final name

**Comment by Emily-Jiang** [comment]
it is still inconsistent as the artifact id still reads MicroProfile.health not McroProfile health check, which is why I suggested to change everything to MicroProfile health

**Pull request merged**